terraform/hydrocron-lambda.tf

```diff
@@ -181,3 +182,23 @@ resource "aws_lambda_permission" "allow_lambda_from_cnm" {
   principal     = "sns.amazonaws.com"
   source_arn    = aws_lambda_function.hydrocron_lambda_cnm.arn
 }
+
+
+resource "aws_lambda_function" "hydrocron_lambda_track_ingest" {
+  package_type = "Image"
+  image_uri    = "${aws_ecr_repository.lambda-image-repo.repository_url}:${data.aws_ecr_image.lambda_image.image_tag}"
+  image_config {
+    command = ["hydrocron.db.track_ingest.track_ingest_handler"]
+  }
+  function_name = local.track_ingest_function_name
+  role          = aws_iam_role.hydrocron_lambda_track_ingest_role.arn
+  timeout       = 300
+  memory_size   = 512
```

Comment:
How confident are we with these limits? Just want to get an idea if we should revisit these later or not.

Reply:
These work well when trying to query all of CMR and then searching Hydrocron for all of the granules located in CMR. I imagine we can reduce them after we get the initial track ingest database populated. Although we would want to increase them for any large "batches" of files that may be queried. I am not quite sure what else would require this besides the initial queries.